src/System.Reflection.Emit/tests/AssemblyBuilder/AssemblyBuilderDefineDynamicModule1.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Threading;
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public class AssemblyGetModuleTests
+    {
+        [Fact]
+        public void PosTest1()
```

Comment:
Filed issue #2183, there are ~700 tests in this assembly, hence tracking as separate workitem.